Assets/MRTK/SDK/Experimental/ScrollingObjectCollection/ScrollingObjectCollection.cs

```diff
@@ -286,80 +291,85 @@ public float AnimationLength
             set { animationLength = value; }
         }
 
-        [Tooltip("Number of items in a row on up-down scroll direction or number of items in a column on left-right scroll direction.")]
+        [Tooltip("Number of cells in a row on up-down scroll view or number of cells in a column on left-right scroll view.")]
         [SerializeField]
         [FormerlySerializedAs("tiers")]
-        private int itemsPerTier = 1;
+        private int cellsPerTier = 1;
 
         /// <summary>
-        /// Number of items in a row on up-down scroll direction or number of items in a column on left-right scroll direction.
+        /// Number of cells in a row on up-down scroll or number of cells in a column on left-right scroll.
         /// </summary>
-        public int ItemsPerTier
+        public int CellsPerTier
         {
             get
             {
-                Debug.Assert(itemsPerTier > 0);
-                return itemsPerTier;
+                Debug.Assert(cellsPerTier > 0);
+                return cellsPerTier;
             }
             set
             {
-                itemsPerTier = value;
+                cellsPerTier = value;
             }
         }
 
+        [Tooltip("Width of the pagination cell.")]
         [SerializeField]
-        [Tooltip("Manual offset adjust the scale calculation of the ClippingBox.")]
-        private Vector3 occlusionScalePadding = new Vector3(0.0f, 0.0f, 0.001f);
-
-        /// <summary>
-        /// Manual offset adjust the scale calculation of the <see cref="Microsoft.MixedReality.Toolkit.Utilities.ClippingBox"/>.
-        /// </summary>
-        /// <remarks>Setting to zero may result in z fighting."</remarks>
-        public Vector3 OcclusionScalePadding
-        {
-            get { return occlusionScalePadding; }
-            set { occlusionScalePadding = value; }
-        }
-
-        [SerializeField]
-        [Tooltip("Manual offset adjust the position calculation of the ClippingBox.")]
-        private Vector3 occlusionPositionPadding = Vector3.zero;
+        private float cellWidth = 0.25f;
 
         /// <summary>
-        /// Manual offset adjust the position calculation of the <see cref="Microsoft.MixedReality.Toolkit.Utilities.ClippingBox"/>.
+        /// Width of the pagination cell.
         /// </summary>
-        public Vector3 OcclusionPositionPadding
+        public float CellWidth
         {
-            get { return occlusionPositionPadding; }
-            set { occlusionPositionPadding = value; }
+            get
+            {
+                Debug.Assert(cellWidth > 0, "Cell width should have a positive non zero value");
```

Comment:
>Debug.Assert(cellWidth > 0, "Cell width should have a positive non zero value"); [](start = 16, length = 80)

same as before - should we move this into set? also clamp the value if we're using it for divison later on?